reference/errors/error_codes.md

```diff
@@ -20,111 +20,285 @@ The requested database has reached its maximum size, and no further documents ca
 
 ## `document_fields_limit_reached`
 
-A document exceeds the maximum limit of 65,535 fields.
+A document exceeds the [maximum limit of 65,535 fields](/learn/advanced/known_limitations.md#maximum-number-of-attributes-per-document).
 
 ## `document_not_found`
 
 The requested document can't be retrieved. Either it doesn't exist, or the database was left in an inconsistent state.
 
-## `dump_not_found`
+## `dump_process_failed`
 
-The requested dump could not be found.  
+An error occurred during the dump creation process. The task was aborted.
 
-## `dump_process_failed`
+## `immutable_api_key_actions`
+
+The [`actions`](/reference/api/keys.md#actions) field of an API key cannot be modified.
+
+## `immutable_api_key_created_at`
+
+The [`createdAt`](/reference/api/keys.md#createdat) field of an API key cannot be modified.
+
+## `immutable_api_key_expires_at`
+
+The [`expiresAt`](/reference/api/keys.md#expiresat) field of an API key cannot be modified.
+
+## `immutable_api_key_indexes`
+
+The [`indexes`](/reference/api/keys.md#indexes) field of an API key cannot be modified.
+
+## `immutable_api_key_key`
+
+The [`key`](/reference/api/keys.md#key) field of an API key cannot be modified.
+
+## `immutable_api_key_uid`
+
+The [`uid`](/reference/api/keys.md#uid) field of an API key cannot be modified.
+
+## `immutable_api_key_updated_at`
+
+The [`updatedAt`](/reference/api/keys.md#updatedat) field of an API key cannot be modified.
+
+## `immutable_index_uid`
 
-An error occurred during dump creation process, task aborted.
+The [`uid`](/reference/api/indexes.md#index-object) field of an index cannot be modified.
 
-## `duplicate_index_found`
+## `immutable_index_updated_at`
 
-The indexes used in the `indexes` array for a [swap index](/reference/api/indexes.md#swap-indexes) request have been declared multiple times. You must declare each index only once.
+The [`updatedAt`](/reference/api/indexes.md#index-object) field of an index cannot be modified.
 
 ## `index_already_exists`
 
-An index with this UID already exists, check out our guide on [index creation](/learn/core_concepts/indexes.md).
+An index with this [`uid`](/reference/api/indexes.md#index-object) already exists, check out our guide on [index creation](/learn/core_concepts/indexes.md).
 
 ## `index_creation_failed`
 
 An error occurred while trying to create an index, check out our guide on [index creation](/learn/core_concepts/indexes.md).
 
+## `index_not_accessible`
+
+An internal error occurred while trying to access the requested index.
+
```

Comment:
I just checked and this error_code doesn't exists anymore, I'm going to remove it from the specification as well.
Thanks for noticing it because I didn't 😅